.github/workflows/npm_publish_dev.yml

```diff
@@ -56,6 +56,6 @@ jobs:
           npm ci
 
       - name: Lerna publish
-        run: npx lerna publish --conventional-commits --conventional-prerelease --conventional-bump-prerelease --create-release github --preid dev --allow-branch develop --force-publish --exact --yes --no-git-tag-version
+        run: npx lerna publish --conventional-commits --conventional-prerelease --conventional-bump-prerelease --create-release github --preid dev --dist-tag dev --allow-branch develop --force-publish --exact --yes --no-git-tag-version
```

Comment:
remove `--conventional-bump-prerelease` and `--no-git-tag-version` - we do not want to artificially bump the dev release version and we also want to tag is as dev